internal/api/shared/authorization/rbac/utils.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rbac
+
+import (
+	"github.com/perses/perses/internal/api/interface/v1/globalrole"
+	"github.com/perses/perses/internal/api/interface/v1/globalrolebinding"
+	"github.com/perses/perses/internal/api/interface/v1/role"
+	"github.com/perses/perses/internal/api/interface/v1/rolebinding"
+	"github.com/perses/perses/internal/api/interface/v1/user"
+	v1 "github.com/perses/perses/pkg/model/api/v1"
+	v1Role "github.com/perses/perses/pkg/model/api/v1/role"
+)
+
+const (
+	GlobalProject = "*"
+)
+
+// UsersPermissions contains the mapping of all users and their permissions
+// username -> project name or global ("") -> permission list
+type UsersPermissions = map[string]map[string][]*v1Role.Permission
+
+func PermissionListHasPermission(permissions []*v1Role.Permission, reqAction v1Role.Action, reqScope v1Role.Scope) bool {
```

Comment:
any reason to have these functions exported ?

Reply:
None, removing